zkevm-circuits/src/rlp_circuit_fsm/util.rs

```diff
@@ -0,0 +1,84 @@
+use eth_types::Field;
```

Comment:
seems this file can also be used by other circuits later? so would it make sense to move it into a more common place?

Reply:
yeah, I can move it to `util` directory.